packages/components/datepicker/README.mdx

```diff
@@ -13,6 +13,14 @@ Short component description covering when it's used.
 
 Optional: Most common misuse and what to use instead.
 
+Props comparison to our current datepickers:
+| Current | DayPicker |
+|---|---|
+| value | selected |
+| onChange | onSelect |
+| min | fromDate |
+| max | toDate |
+
```

Comment:
This doesn't work well on the website, maybe we can do a proper markdown table instead?

<img width="767" alt="Screenshot 2022-06-13 at 15 42 49" src="https://user-images.githubusercontent.com/219017/173367306-f7235c4f-a3ed-4c9e-ade8-805eed14220f.png">

Reply:
I believe @mshaaban0 will adjust it while working on docs.